### ISSUE TITLE
Help wanted: unknown attribute #[wasm_bindgen(start)]

### ISSUE THREAD

**Issue by cmac4603** [rustwasm/wasm-bindgen#1075]
Is someone able to explain why if I run `./build.sh` in an example (let's say `dom`), it runs fine locally in this repo.

But if I copy the full example directory `dom` somewhere else on my computer, I get the following error:
```rust
+ cargo build --target wasm32-unknown-unknown --verbose
       Fresh unicode-xid v0.1.0
       Fresh version_check v0.1.5
       Fresh cfg-if v0.1.6
       Fresh lazy_static v1.2.0
       Fresh rustc-demangle v0.1.9
       Fresh ucd-util v0.1.3
       Fresh unicode-segmentation v1.2.1
       Fresh quick-error v1.2.2
       Fresh utf8-ranges v1.0.2
       Fresh termcolor v1.0.4
       Fresh sourcefile v0.1.4
       Fresh log v0.4.6
       Fresh thread_local v0.3.6
       Fresh regex-syntax v0.6.4
       Fresh heck v0.3.0
       Fresh humantime v1.2.0
       Fresh proc-macro2 v0.4.24
       Fresh libc v0.2.44
       Fresh wasm-bindgen-shared v0.2.28
       Fresh quote v0.6.10
       Fresh backtrace v0.3.9
       Fresh atty v0.2.11
       Fresh memchr v2.1.1
       Fresh syn v0.15.22
       Fresh aho-corasick v0.6.9
       Fresh nom v4.1.1
       Fresh synstructure v0.10.1
       Fresh wasm-bindgen-backend v0.2.28
       Fresh regex v1.1.0
       Fresh weedle v0.8.0
       Fresh failure_derive v0.1.3
       Fresh wasm-bindgen-macro-support v0.2.28
       Fresh env_logger v0.6.0
       Fresh failure v0.1.3
       Fresh wasm-bindgen-macro v0.2.28
       Fresh wasm-bindgen-webidl v0.2.22
       Fresh wasm-bindgen v0.2.28
       Fresh js-sys v0.3.5
       Fresh web-sys v0.3.5
   Compiling dom v0.1.0 (/Users/cmacrae/gitrepos/dom)
     Running `rustc --crate-name dom src/lib.rs --color always --crate-type cdylib --emit=dep-info,link -C debuginfo=2 -C metadata=786d048b9132a370 --out-dir /Users/cmacrae/gitrepos/dom/target/wasm32-unknown-unknown/debug/deps --target wasm32-unknown-unknown -C incremental=/Users/cmacrae/gitrepos/dom/target/wasm32-unknown-unknown/debug/incremental -L dependency=/Users/cmacrae/gitrepos/dom/target/wasm32-unknown-unknown/debug/deps -L dependency=/Users/cmacrae/gitrepos/dom/target/debug/deps --extern wasm_bindgen=/Users/cmacrae/gitrepos/dom/target/wasm32-unknown-unknown/debug/deps/libwasm_bindgen-0b44cf352427bf84.rlib --extern web_sys=/Users/cmacrae/gitrepos/dom/target/wasm32-unknown-unknown/debug/deps/libweb_sys-79369d3dddd365f2.rlib`
error: unknown attribute
 --> src/lib.rs:7:16
  |
7 | #[wasm_bindgen(start)]
  |                ^^^^^

error: aborting due to previous error

error: Could not compile `dom`.

Caused by:
  process didn't exit successfully: `rustc --crate-name dom src/lib.rs --color always --crate-type cdylib --emit=dep-info,link -C debuginfo=2 -C metadata=786d048b9132a370 --out-dir /Users/cmacrae/gitrepos/dom/target/wasm32-unknown-unknown/debug/deps --target wasm32-unknown-unknown -C incremental=/Users/cmacrae/gitrepos/dom/target/wasm32-unknown-unknown/debug/incremental -L dependency=/Users/cmacrae/gitrepos/dom/target/wasm32-unknown-unknown/debug/deps -L dependency=/Users/cmacrae/gitrepos/dom/target/debug/deps --extern wasm_bindgen=/Users/cmacrae/gitrepos/dom/target/wasm32-unknown-unknown/debug/deps/libwasm_bindgen-0b44cf352427bf84.rlib --extern web_sys=/Users/cmacrae/gitrepos/dom/target/wasm32-unknown-unknown/debug/deps/libweb_sys-79369d3dddd365f2.rlib` (exit code: 1)
```

I am reasonably confident the toolchains are the same, with both using stable by default with no local overrides.

Apologies I know this isn't an "issue", but unsure where else to ask and I have been banging my head against this for a little while.

**Comment by limira** [comment]
`#[wasm_bindgen(start)]` is only available in git repo, not in `wasm_bindgen v0.2.28`. You must use `git` or wait for `wasm_bindgen v0.2.29` to use `#[wasm_bindgen(start)]`

**Comment by cmac4603** [comment]
I see! Thank you @limira 

Actually relieved I'm not going mad :)
